site/kube-addon.md

```diff
@@ -98,7 +99,7 @@ and so you will need to perform the procedure manually:
 * Apply the updated addon manifest `kubectl apply -f https://git.io/weave-kube`
 * Kill each Weave Net pod with `kubectl delete` and then wait for it to reboot before moving on to the next pod.
 
-**Note:** If you delete all Weave Net pods at the same time they will
+**Note:** For Weave Net versions prior to 2.0, if you delete all Weave Net pods at the same time they will
   lose track of IP address range ownership, possibly leading to
```

Comment:
In versions prior to Weave Net 2.0, deleting all of the Weave Net pods at the same time will result in them losing track of IP address range ownership, ....